content/en/docs/reference/command-line-tools-reference/kube-controller-manager.md

```diff
@@ -460,7 +460,7 @@ kube-controller-manager [flags]
 <td colspan="2">--feature-gates &lt;comma-separated 'key=True|False' pairs&gt;</td>
 </tr>
 <tr>
```

Comment:
Don't include this file. The file is auto-generated. We'll update the whole file when 1.24 is out.

Reply:
Sure. I will fix this.